packages/backend-core/src/Base.ts

```diff
@@ -68,10 +70,17 @@ export class Base {
      * 2. Use load function
      * 3. Try to load from env
      */
-    const availableKey =
-      !jwtKey && this.loadCryptoKeyFunction
-        ? await this.loadCryptoKeyFunction(token)
-        : await this.loadCryptoKey(jwtKey || process.env.CLERK_JWT_KEY);
+
+    let availableKey: CryptoKey;
+    if (!jwtKey && this.loadCryptoKeyFunction) {
+      try {
+        availableKey = await this.loadCryptoKeyFunction(token);
+      } catch (_) {
+        throw new TokenVerificationError(TokenVerificationErrorReason.PublicKeyFetchError);
```

Comment:
Is it possible to end up here if `CLERK_API_KEY` is not set?

If so, can we differentiate between `KeyMissing` and `PublicKeyFetchError`?